v3data/hypes/fees.py

```diff
@@ -267,4 +269,30 @@ def calc_fees(
             liquidity * (sub_in_256(fees_accum_now_1, fee_growth_inside_last_1))
         ) / X128
 
-        return uncollectedFees_0, uncollectedFees_1
+        # apply filter to result
+        return Fees.calc_fees_filter(uncollectedFees_0, uncollectedFees_1)
```

Comment:
Can we leave the filter to be applied downstream?

Here:
https://github.com/GammaStrategies/uniswap-v3-performance/blob/f290020a6c8b0da4acd53b04f0ed457a0113d96f/v3data/hypes/fees_yield.py#L58

a filter is applied based on the max yield per day, and sets the hasOutlier field.  This functionality would be lost if the filter is applied to uncollected fees already.

Reply:
Ok. so if the calculated daily yield is greater than X, we show the has_outlier field True, without changing results. Clean.
So I will delete this filter